docs/_layouts/global.html

```diff
@@ -73,10 +73,15 @@
       </div>
       </div>
 
-      <div class="col-9" id="content">
+      <div class="col-6" id="content">
```

Comment:
shrinking this to 2/3 the original size seems to be a bit narrow. we can merge this for now but i think we should consider making the other sections more narrow too to make more space for the content. the font size and spacing may need to be reduced for the two side bars